### DISCUSSION TITLE
Sensor not triggering on job success

### DISCUSSION THREAD

**Discussion by Swejal08** [dagster-io/dagster#19987] [Q&A]
Using dagster version 1.6.0

__init__.py
```
all_sensors = [new_files_sensor, document_processing_on_job_success]

defs = Definitions(
    assets=all_assets,
    sensors=all_sensors,
    jobs=[extract_and_load],
)
```

I have a job that runs some assets

jobs.py
```
extract_and_load = define_asset_job(
    name="extract_and_load",
    selection=[
        "download_file_from_s3",
        "extract_text",
        "load_vector_embedding",
    ],
    hooks={document_processing_failure_hook, document_processing_success_hook},
)
```

When this job succeeds, i want to run a sensor to do some tasks. For this i am using `run_status_sensor`

run_status_sensor.py
```
@run_status_sensor(monitored_jobs=[extract_and_load],  run_status=DagsterRunStatus.SUCCESS)
def document_processing_on_job_success(context: RunStatusSensorContext):
    document_id = context.partition_key

    context.log.info(
        f"Running job success sensor for: {document_id}"
    )

    update_document_status(
        document_id, context.log, DocumentLoadingStatus.LOADED.value
    )

    context.log.info(f"Marked document {document_id} as LOADED on job success")
```
    
   The problem is even after my job is successful, this sensor wont run .
   Any clue on what am i exactly missing here  ?

**Comment by garethbrickman** [comment]
A couple things to try/check: 

1. Check Sensor Configuration: Ensure that the monitored_jobs parameter is correctly referencing the job you want to monitor. It should be the name of the job as a string, not the job object itself. If extract_and_load is a variable representing the job, you should pass `monitored_jobs=["extract_and_load"]`.

2. Check Sensor Status: Make sure that the sensor is in an active state. Sensors can be paused, and if your sensor is not active, it will not run. You can check the status of your sensor in the Dagster UI or via the DagsterInstance API.

3. Check Sensor Evaluation Interval: Sensors have a minimum interval between evaluations. If your job completes but the sensor does not run, it might be due to the sensor's evaluation interval not having elapsed yet. You can adjust the minimum_interval_seconds parameter if needed.
